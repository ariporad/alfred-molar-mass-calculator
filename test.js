import test from 'ava';
import alfyTest from 'alfy-test';

const tests = ['NaOH', 'H2O', 'H2O2', 'CO2', 'CO', 'NaCl'];

async function macro(t, input) {
	const alfy = alfyTest();
	const result = await alfy(input);

	t.snapshot(result);
}
macro.title = (providedTitle, input) => input;

tests.forEach(input => test(macro, input));
