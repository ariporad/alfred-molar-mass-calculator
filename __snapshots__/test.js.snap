exports[`CO 1`] = `"[{\"title\":\"The Molar Mass of CO is 28.010 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"28.010\"}]"`;

exports[`CO2 1`] = `"[{\"title\":\"The Molar Mass of CO2 is 44.009 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"44.009\"}]"`;

exports[`H2O 1`] = `"[{\"title\":\"The Molar Mass of H2O is 18.015 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"18.015\"}]"`;

exports[`H2O2 1`] = `"[{\"title\":\"The Molar Mass of H2O2 is 34.015 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"34.015\"}]"`;

exports[`NaCl 1`] = `"[{\"title\":\"The Molar Mass of NaCl is 58.441 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"58.441\"}]"`;

exports[`NaOH 1`] = `"[{\"title\":\"The Molar Mass of NaOH is 39.997 g/mol\",\"subtitle\":\"Press Enter to Copy\",\"arg\":\"39.997\"}]"`;
